plugins/content/createmenulink/createmenulink.php

```diff
@@ -163,7 +164,9 @@ public function onContentAfterSave($context, $article)
 		$data    = $session->get("formData");
 		$session->clear("formData");
 
-		list($component, $view) = explode('.', $context);
+		$jinput    = JFactory::getApplication()->input;
+		$component = $jinput->get('option');
+		$view      = $jinput->get('view');
```

Comment:
I think it is because url doesn't have view in option. Did you tried to get data from post instead of request?